testing/citest/tests/google_http_lb_upsert_scenario.py

```diff
@@ -337,17 +345,20 @@ def delete_http_load_balancer(self):
     (contract_builder.new_clause_builder('Health Check Removed',
                                          retryable_for_secs=30)
      .list_resource('httpHealthChecks')
-     .excludes_path_value('name', self.__proto_hc['name'])
+     .EXPECT(ov_factory.value_list_path_excludes(
```

Comment:
I like these -- seems like with this structure we can iterate on the expressiveness.